### DISCUSSION TITLE
New Wave Updates 🌊 [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#62081] [New to GitHub]
Popping by to end the month with a great round up of shout outs and the latest from GitHub. Let’s jump in!

🌟**Community Superstars**: A round of applause for these community citizens 👏 

* @yaxan: For sharing their patience and knowledge with us on a post about [restoring a forked repository](https://github.com/orgs/community/discussions/60775). 
* @L9us: Has been lending a hand all over the community, but I love their response on the [cloning Discussion](https://github.com/orgs/community/discussions/61556) which included a helpful screenshot example. Nicely done! 
* @ramamimu: For sharing how to automatically change your default name to main instead of master when creating a new git initialization in [their comment](https://github.com/orgs/community/discussions/60619#discussioncomment-6458624). 
 
 💫 Want to nominate a community member for a feature above? Just tag me         (@queenofcorgis) where you’ve seen great community behavior. 💫

🆕 **What’s new?** The latest news and releases from GitHub. 

* **Troubleshooting 404 errors for GitHub Pages sites guide**: Based on community posts and solutions, we compiled solutions to common errors so you can easily solve yours and help your fellow community members solve theirs ❣️. [View the guide](https://github.com/orgs/community/discussions/61073) or bookmark it for later. 
* **GitHub Repository Rules are now generally available**: Repository rules provide an easy, flexible way to define branch protections and ensure consistency in code across repositories. @patrick-knight shares our new approach to repository rules [here](https://github.blog/2023-07-24-github-repository-rules-are-now-generally-available/). 
* **Improved Git error message for broken repositories**: We’ve upgraded the message to be more informative, check out the details [on our Changelog](https://github.blog/changelog/2023-07-19-improved-git-error-message-for-broken-repositories/).  

📚 **Rousing Reads and Beyond**: GitHub-produced goodies 🍭
* **You might not need... bootcamp!**: @jamieshark breaks down their nontraditional journey to software engineering with their [blog post](https://github.com/orgs/community/discussions/61190) 🥾. 
* **A developer’s guide to prompt engineering and LLMs**: Prompt engineering is the art of communicating with a generative AI model. [In this article](https://github.blog/2023-07-17-prompt-engineering-guide-generative-ai-llms/), @wunderalbert & @jnbrymn cover how we approach prompt engineering at GitHub, and how you can use it to build your own LLM-based application.

Have you learned something new on GitHub this month? Share it below.

**Comment by Lodoelama** [comment]
This month I've learned a lot about Malware Analysis and GitHub has been a tremendous resource to find write ups and malware samples. Here is the GitHub for the course I'm taking if anyone's interested https://github.com/HuskyHacks/PMAT-labs

**Comment by mehulrajdev** [comment]
Check this please

https://github.com/mehulrajdev/Zayn-Malik](https://github.com/mehulrajdev/Zayn-Malik)

https://github.com/mehulrajdev

**Reply by queenofcorgis** [reply]
Hi Mehul - if you're looking for help on a project or have a specific question, I recommend starting a new discussion as that is more likely to get visibility.

**Comment by queenofcorgis** [comment]
We've got a fresh crop of superstars - join me in thanking them in my [new check-in](https://github.com/orgs/community/discussions/63401).
